sympy/functions/special/bsplines.py

```diff
@@ -254,6 +268,17 @@ def interpolating_spline(d, x, X, Y):
     Piecewise((7*x**3/117 + 7*x**2/117 - 131*x/117 + 2, (x >= -2) & (x <= 1)),
             (10*x**3/117 - 2*x**2/117 - 122*x/117 + 77/39, (x >= 1) & (x <= 4)))
 
+    Parameters
+    ==========
+
+    d : integer
+    Degree of Bspline strictly greater than equal to one
```

Comment:
Does this need to be indented to render properly?

Reply:
Done!